.github/actions/build-and-test/action.yml

```
@@ -0,0 +1,69 @@
+      run: pnpm lint:check
+
+    - name: Run unit tests with coverage
+      if: inputs.skip-tests == 'false'
+      shell: bash
+      env:
+        JWT_SECRET: test-secret
+        JWT_REFRESH_SECRET: test-refresh-secret
+        NODE_ENV: test
+      run: |
+        pnpm --filter @bluelight-hub/backend test:cov
+        pnpm --filter @bluelight-hub/frontend test:cov
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)